docs_source_files/content/grid/grid_4/graphql_support.de.md

```diff
@@ -81,18 +107,18 @@ The best way to query GraphQL is by using `curl` requests. GraphQL allows you to
 
 Some of the example GraphQL queries are given below. You can build your own queries as you like.
 
-### Querying the number of `totalSlots` and `usedSlots` in the grid :
+### Querying the number of `maxSession` and `sessionCount` in the grid :
```

Comment:
please let me know if you think we should keep `totalSlots` in this example, since it's in the schema.

I wasn't sure how to correlate the two since `usedSlots` got removed

Reply:
No need to keep it, `maxSession` and `sessionCount` are perfect here.